starter/script.js

```diff
@@ -1 +1,94 @@
 // put code here!
+var name=prompt('What is your name?')
+ // ask which quest to discover
+ var quest=prompt('What quest would you like to go on? Save the Empress or Find the treasure? (e/t) ')
+
+ if(prompt=="e"){
```

Comment:
want to check the value `quest` here so `if (quest === "e")`. We are using the prompt to ask the question, and then the response is being assigned to the variable `quest`